### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by tylerschroeder7** [tylerschroeder7/MI-449-SS17-740-css-stealing-styles-xedvDK#1]
@stuartpearman Can you take a look at this? It's [hosted here](https://tylerschroeder7.github.io/MI-449-SS17-740-css-stealing-styles-xedvDK/) and meets the following criteria:

- [x] The page includes the name of your search engine in big letters (you can make it up), with a search `input` below it, then a button below that to run the search (the button doesn't have to do anything for this prototype)
- [x] The font size of the search engine name is at least 70 pixels, so that it's very easy to see
- [x] `span` elements are wrapped around each letter of the name, like this: `<span>G</span><span>o</span><span>o</span><span>g</span><span>l</span><span>e</span>`, so that they can be styled individually with classes
- [x] The on-hover colors from the paragraphs [on this page](http://skillful-clouds.surge.sh/) are used to color individual letters of your name (similar to the letters in Google's logo)
- [x] Inspect the `input` element at the bottom of the [_Prototyping states_ page of the lesson](5) to create an identical-looking search box for your own site (it should even use the same styles on `focus`)
- [x] The styles from the "Create Account" button [on this page](http://plant-servant.surge.sh/) are used for your "Search" button (it should even use the same styles on `hover` and `active`)
- [x] There are no stylelint errors in your CSS (this will also be expected for all future projects)
- [x] All your CSS exists in a `.css` file

**Comment by KatieMFritz** [comment]
Hi Tyler, this is really close! Great job on the site name and the `input` box.

The search button doesn't quite match yet - I'm seeing two different shades of blue in your version vs. one in the source, and also the source uses a sans-serif font.

Let me know if you have any questions. Otherwise, comment back here once you've made changes and checked your hosted site! 😁

**Comment by tylerschroeder7** [comment]
Okay, it should be good now. I was seeing three different blues for active, hover, and normal so I just used the eyedropper.

**Comment by KatieMFritz** [comment]
<img src="https://www.snoopn4pnuts.com/images/products/kodomo007.jpg" width="200" alt="good job snoopy">

One thing I forgot to check before! Right now, you are using class names for your "Snoopy" letters that include the letter name, like `purpleO`. This is valid CSS, but in general, it's more flexible to not describe your specific content in your class names. That way, if you want to assign the class to a different piece of content, or reuse them for more than one piece of content, it still makes sense.

In your case, I would just use the color names for your classes. Sound good? 😄

**Comment by tylerschroeder7** [comment]
Yeah, that makes sense. I thought about doing that but for some reason decided to use a specific class for each letter haha. I'll remember for next time though. Thank you!

**Comment by tylerschroeder7** [comment]
Do you want me to change it and submit it again?

**Comment by KatieMFritz** [comment]
Yes please - that way, if you look back to reference this project in the future, it will be right. Sorry for not making that more clear!

**Comment by tylerschroeder7** [comment]
Should be good now!

**Comment by KatieMFritz** [comment]
<img src="https://s-media-cache-ak0.pinimg.com/736x/80/73/aa/8073aa20588ec6c820c25fc517420ea2.jpg" alt="happy snoopy"  width="200">

:shipit:
